planet/cli/data.py

```diff
@@ -295,6 +342,32 @@ async def search_create(ctx, name, item_types, filter, daily_email, pretty):
         echo_json(items, pretty)
 
 
+@data.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@click.argument("item_types", callback=parse_item_types)
+@click.argument('interval',
+                default=None,
```

Comment:
I believe cl.get_stats will fail if stats_interval is None. I recommend removing the default value. Is there a test for the CLI that would tell us if not specifying interval would result in an error or not?